config/README.md

```diff
@@ -38,33 +38,35 @@ configuration from many of the services.
 
 ## Building a Test Network (TestNet)
 
-`config-builder` builds an entire network's or swarm's configuration including the
-genesis blob. It takes as one of its input parameters an index that selects, which
-of these configs to return. This can be used to create a Libra TestNet by specifying
-distinct indices for each validator. Similarly the tool can be used to add Fullnodes
-to an existing network.  Finally, it enables generation of a mint/faucet client
-capable of performing mint transactions/creating accounts.
+`config-builder` builds an entire configuration for a Validator or FullNode, 
+including the genesis blob. It takes as one of its input parameters an index that
+specifies the specific node config to return. This can be used to create a Libra 
+TestNet by constructing compatible configurations the full set of Validators.
```

Comment:
missing a preposition after configurations and before the full set 

could imagine for or across

Reply:
done.